### PR TITLE
Improve error messages for people trying to install the CLI with Python2

### PR DESCRIPTION
**Pull request by wilzbach**
This comes from a request from the public slack as people just tried using `pip install --user story` on Ubuntu. It still defaults to Python2 there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,19 @@ The goals of this project is to provide a utility for developers to interact wit
...
+```
+
+Python 3.6 or higher is required.
+On Ubuntu/Debian, use `pip3`.
+
+### OSX
+
 ```shell
-$ brew install storyscript/brew/story
+brew install storyscript/brew/story
```
Comment (wilzbach) [inline README.md]: This was done for consistency as the other shell commands in this README are without `$`.
I don't have a strong opinion about this, but we should stay consistent.

Comment (kenneth-reitz) [reply README.md]: i agree

**Comment by kenneth-reitz** [comment]
I'm torn about this. It will break tools like pipenv.

**Comment by kenneth-reitz** [comment]
setup.py should always execute.

**Comment by kenneth-reitz** [comment]
print, don't exit. rely on requires_python.

**Comment by wilzbach** [comment]
Hmm, but Python2 doesn't support the `requires_python`, so the user is left with no clue about why the installation failed.

**Pull request merged**